### ISSUE TITLE
add "next/previous month" button on calendar webpage even in day-view

### ISSUE THREAD

**Issue by kollix** [nextcloud/calendar#554]
It would be nice when the web calendar would also allow one to skip to next/previous month even when I'm just in day or wekk-view.
Clicking into the actual day entry field pops down a mini-calendar where I can quickly navigate to a specific day in a month, which is great, but what is missing here is that I also can switch to another month in a quick way.

**Comment by georgehrke** [comment]
Yes, when the mini-calendar is open, it should always use steps of one month

cc @jancborchardt
